### DISCUSSION TITLE
Change Poison Button Focus and click color

### DISCUSSION THREAD

**Discussion by behroozbc** [Taiizor/ReaLTaiizor#85] [Q&A] [answered]
Hi, I want to change Poison Button Focus and click color. But, I can not find related settings. The default color is gray.

An example of the focus and click color: ![An example of the focus and click color](https://user-images.githubusercontent.com/20381616/201922959-c7fe9845-97cf-4622-8725-3c9487f8965a.gif)

**Comment by Taiizor** [comment, marked as answer]
There is a component that manages the theme of the poison theme. The name of the component is PoisonStyleManager. Therefore, some colors can be customized, while others are automatically selected according to the Light or Dark theme.

You can see colors that can't be customized in the file below.
https://github.com/Taiizor/ReaLTaiizor/blob/develop/src/ReaLTaiizor/Drawing/Poison/PoisonPaint.cs#L30

**Reply by behroozbc** [reply]
It is possible to change it, as I found.
